### PR TITLE
fix: homepage

### PR DESCRIPTION
**Pull request by ksted**
- Alt text for hero images
- Product carousel
- Demo data

## Why:

Closes: #ID

## Checklist before requesting a review

- [x] My code follows the code style of this project.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [x] I've updated `docs/changelog/changelog_en.md` and `docs/changelog/changelog_de.md`. If I'm a non-German speaker, I've still updated the file with the English version as a placeholder.

### CODE REVIEW

File: apps/web/components/ui/VsfLogo.vue [hunk 1]
```diff
@@ -1,13 +1,21 @@
 <template>
   <picture>
     <template v-if="imageExtension === 'svg'">
-      <NuxtImg :src="imagePath" alt="Logo" class="w-100 h-10 py-2" width="150" height="40" preload />
+      <NuxtImg
+        :src="imagePath"
+        :alt="`${{ storeName }} logo`"
```
Comment (aoltean-plenty) [inline apps/web/components/ui/VsfLogo.vue:6]: I think you should use the binding syntax here: 

`        :alt="`${storeName} logo`"`

File: apps/web/components/ui/VsfLogo.vue [hunk 2]
```diff
@@ -1,13 +1,21 @@
...
+        ref="logo"
+        preload
+      />
     </template>
     <template v-else>
       <img
         id="logo"
         :src="imagePath"
-        alt="Header Logo"
+        :alt="`${{ storeName }} logo`"
```
Comment (aoltean-plenty) [inline apps/web/components/ui/VsfLogo.vue:18]: Same:

        :alt="${storeName} logo"

File: apps/web/composables/useHomepageData/useHomepageData.ts [hunk 3]
```diff
@@ -1,17 +1,21 @@
-import { ref, watch, computed } from 'vue';
 import { HeroContentProps, SizeKey } from '~/components/ui/HeroCarousel/types';
 import { MediaItemProps } from '~/components/ui/MediaCard/types';
-import { useCategoryTree, useCategoryTemplate } from '~/composables';
-import homepageTemplateData from './homepageTemplateData.json';
+import { useCategoryTemplate } from '~/composables';
```
Comment (aoltean-plenty) [inline apps/web/composables/useHomepageData/useHomepageData.ts]: Nuxt scope should recognize the composable. You don't need the import.

**Review by aoltean-plenty — CHANGES_REQUESTED** [review]

File: apps/web/components/ui/VsfLogo.vue (hunk 1 above)
Comment (ksted) [reply apps/web/components/ui/VsfLogo.vue:6]: This syntax throws a parsing error for me.

File: apps/web/composables/useHomepageData/useHomepageData.ts (hunk 3 above)
Comment (ksted) [reply apps/web/composables/useHomepageData/useHomepageData.ts]: Done.

**Review by aoltean-plenty — APPROVED** [review]

**Pull request merged**